### PR TITLE
feat: update footer with data portal logo (#3752)

### LINKED ISSUE #3752: [HCA DX] Update footer with data portal logo
See [mock](https://www.figma.com/file/i4uZsYkKAML9ivi2TIo6ug/HCA-Data-Explorer?type=design&node-id=1148-18089&mode=design&t=0qDgaYzw3yBsffSq-4)

### PR DESCRIPTION
**Pull request by frano-m**
### Ticket
Closes #3752.

### Reviewers
@NoopDog.

### Changes
- Updated footer with data portal logo.
- Updated `@clevercanary/data-explorer-ui`  to `0.54.0`.

### CODE REVIEW

File: explorer/site-config/anvil-cmg/cc-dev/config.ts [hunk 1]
```diff
@@ -3,7 +3,10 @@ import { makeConfig } from "../dev/config";
 import { authenticationConfig } from "./authentication/authentication";
 
 const config: SiteConfig = {
-  ...makeConfig("https://anvilproject.dev.clevercanary.com"),
+  ...makeConfig(
+    "https://anvilproject.dev.clevercanary.com",
+    "https://anvilproject.dev.clevercanary.com"
```
Comment (frano-m) [inline explorer/site-config/anvil-cmg/cc-dev/config.ts:8]: @NoopDog added PORTAL_URL to the configuration.

File: explorer/site-config/anvil-cmg/dev/config.ts [hunk 2]
```diff
@@ -99,14 +95,13 @@ export function makeConfig(
...
     dataSource: {
-      defaultDetailParams: {
-        catalog: catalog,
-      },
       defaultListParams: {
-        catalog: catalog,
         size: "25",
         sort: "entryId",
       },
+      defaultParams: {
```
Comment (frano-m) [inline explorer/site-config/anvil-cmg/dev/config.ts:102]: @NoopDog catalog now under `defaultParams`.

File: explorer/site-config/anvil-cmg/prod/config.ts [hunk 3]
```diff
@@ -7,7 +7,11 @@ import { downloadColumn } from "../dev/index/filesEntityConfig";
 import { authenticationConfig } from "./authentication/authentication";
 
 const config: SiteConfig = {
-  ...makeConfig("https://prod.anvil.gi.ucsc.edu", "anvil3"),
+  ...makeConfig(
+    "https://prod.anvil.gi.ucsc.edu",
+    "https://prod.anvil.gi.ucsc.edu",
```
Comment (frano-m) [inline explorer/site-config/anvil-cmg/prod/config.ts:12]: PROD PORTAL_URL

File: explorer/site-config/hca-dcp/dev/config.ts [hunk 4]
```diff
@@ -18,12 +18,10 @@ import { samplesEntityConfig } from "./index/samplesEntityConfig";
...
 import { supportConfig } from "./support/support";
 
-const hcaExplorer = "/images/hcaExplorer.png";
-const logoHumanCellAtlas = "/images/logoHumanCellAtlas.png";
-
 // Template constants
 const APP_TITLE = "HCA Data Explorer";
 const BROWSER_URL = "https://data.humancellatlas.dev.clevercanary.com";
+const ORG_URL = "https://www.humancellatlas.org";
 const PORTAL_URL = "https://data.humancellatlas.dev.clevercanary.com";
```
Comment (frano-m) [inline explorer/site-config/hca-dcp/dev/config.ts:25]: DEV PORTAL_URL.

File: explorer/site-config/hca-dcp/prod/config.ts [hunk 5]
```diff
@@ -5,6 +5,7 @@ import devConfig from "../dev/config";
 
 // Template constants
 const BROWSER_URL = "https://data.humancellatlas.org";
+const ORG_URL = "https://www.humancellatlas.org";
 const PORTAL_URL = "https://data.humancellatlas.prod.clevercanary.com";
```
Comment (frano-m) [inline explorer/site-config/hca-dcp/prod/config.ts:9]: PROD PORTAL_URL

**Review by NoopDog — APPROVED** [review]

**Pull request merged**